home/tests/test_api.py

```diff
@@ -94,16 +153,41 @@ def create_content_page(
 
         bodies = []
 
+        if image and Image.objects.all().count() == 0:
+            mk_test_img()
+        if media and Media.objects.all().count() == 0:
+            mk_test_media()
+        if doc and Document.objects.all().count() == 0:
+            mk_test_doc()
+
+        image_id = Image.objects.first().id if image else None
+        media_id = Media.objects.first().id if media else None
+        doc_id = Document.objects.first().id if doc else None
+
         for i in range(body_count):
             msg_body = f"*Default {body_type} Content {i+1}* 🏥"
             if body_type == "WhatsApp":
-                bodies.append(WABody(title, [WABlk(msg_body)]))
+                bodies.append(
+                    WABody(
+                        title,
+                        [
+                            WABlk(
+                                msg_body,
+                                image=image_id,
+                                media=media_id,
```

Comment:
We could probably save some vertical space by extracting the `WABlk(...)` to a local variable and passing it to `WABody(...)`.